src/NetMQ/NetMQFrame.cs

```diff
@@ -278,6 +278,33 @@ public override bool Equals(object obj)
         }
 
         /// <summary>
+        /// Return true if this one and the other NetMQFrame are equal, or both are null.
+        /// </summary>
+        /// <param name="one">one frame to compare against the other</param>
+        /// <param name="other">the other frame to compare</param>
+        /// <returns>true if both frames are equal</returns>
+        public static bool operator ==(NetMQFrame one, NetMQFrame other)
```

Comment:
I think implementing the == operator is a little problematic.

First it might break existing systems, when you want to make sure it is equals you use .Equals and == for reference equality